Cargo.toml

```diff
@@ -22,7 +22,7 @@ itertools = { version = "0.12.1", default-features = false }
 unicode-width = "0.1"
 
 [dev-dependencies]
-criterion = "0.5"
+criterion = "0.5.1"
```

Comment:
Same as [my other comment](https://github.com/Aetf/unicode-truncate/pull/10#discussion_r1586209894) even more extreme: the actual dependency version stays the same, only the Cargo.toml gets more verbose.